app/Models/Extensions/PhotoCast.php

```diff
@@ -84,6 +84,7 @@ public function toReturnArray(): array
 			'taken_at_orig_tz' => $this->taken_at_orig_tz,
 			'license' => $this->license,
 			'sizeVariants' => $sizeVariants,
+			'colors' => $this->colors()->get(),
```

Comment:
`$this->colors` is enough and more efficient. (Note that the missing parentheses `()` at the end are by intention.) This way you call the attribute `colors` (which is a `Collection` of `Color`) and not the function which always returns a `HasMany` object. The difference is that the attribute uses caching while `colors()->get()` always queries the database.